Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM adoptopenjdk/openjdk11:alpine-jre
+WORKDIR /opt/app
+COPY build/libs/ContactsManager-0.0.1-SNAPSHOT.jar ContactsManager-0.0.1.jar
+CMD ["java", "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=*:5005", "-jar", "ContactsManager-0.0.1.jar"]
```

Comment:
Expose port 8080 & use entry point to stop parameters overriding through command line.

Reply:
@HattoriHenzo

> Did  you need more time or i complete this ??

Reply:
I will post a new PR very soon

Reply:
@HattoriHenzo then, i am closing this one.